### DISCUSSION TITLE
Publication ID for older TED_EXPORT

### DISCUSSION THREAD

**Discussion by poetsmeniet** [OP-TED/ted-xml-data-converter#25] [Q&A] [answered]
Hi cool guys of OP-TED

I noticed for older documents (2019) I get the following result for each document:

"EFAC:PUBLICATION": {
  "EFBC:NOTICEPUBLICATIONID": {
                "_text": "12345678-2023"
              }

Is this a bug or are older documents not supported?

**Comment by pdonohoe** [comment, marked as answer]
Hi @poetsmeniet,
This is a bug / oversight. The code for this has not yet been written. But the dummy ID above is hard-coded, so it should exist for every document, not just older ones.
It will be fixed in a forthcoming release of the converter.

**Reply by poetsmeniet** [reply]
Thanks Paul. Can you indicate a time frame?

Is there any way I could assist?

**Comment by pdonohoe** [comment]
I'm sorry, I can't indicate a timeframe. Although the fix is relatively simple, I don't manage the deployments. We are about to deploy a new version of the converter, which unfortunately doesn't contain the fix. I don't know when we can deploy the version after.
